jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileCredentialProviderFactoryTest.kt

```diff
@@ -745,6 +759,69 @@ class ProfileCredentialProviderFactoryTest {
         }
     }
 
+    @Test
+    fun `sso-session profile creates a provider`() {
+        writeProfileFile(
+            """
+            [profile sso]
+            sso_session = my-sso
+            sso_account_id=111222333444
+            sso_role_name=RoleName
+            
+            [sso-session my-sso]
+            sso_region=us-east-2
+            sso_start_url=ValidUrl
+            sso_registration_scopes = sso:validAcc:validAccess
+            """.trimIndent()
+        )
+
+        clientManager.create<SsoClient>()
+        clientManager.create<SsoOidcClient>()
+
+        val providerFactory = createProviderFactory()
+        val validProfile = findCredentialIdentifier("sso")
+        val credentialsProvider = providerFactory.createProvider(validProfile)
+        assertThat(credentialsProvider).isInstanceOf<ProfileSsoSessionProvider>()
```

Comment:
add checks that the values are deserialized properly

Reply:
Added those